src/components/Authentication/Authentication.vue

```diff
@@ -0,0 +1,94 @@
+<template lang="pug">
+v-flex.pa-2.girder-authentication-component(xs-12)
+  v-card
+    v-tabs(v-model="activeTab", color="primary", slider-color="yellow", dark)
+      v-tab(key="login") Log In
+      v-tab(key="registration", v-if="register") Register
+      v-tab-item(key="login-box")
+        login(ref="loginForm",
+            :oauth-providers="oauthProviders",
+            :forgot-password-url="forgotPasswordUrl",
+            :forgot-password-route="forgotPasswordRoute",
+            @forgotpassword="$emit('forgotpassword')")
+      v-tab-item(key="registration-box", v-if="register")
+        registration(
+            ref="registerForm",
+            :oauth-providers="oauthProviders")
+</template>
+
+<script>
+import Login from './Login.vue';
+import Registration from './Register.vue';
+
+export default {
+  inject: ['girderRest'],
+  props: {
+    register: {
+      type: Boolean,
+      default: false,
+      note: 'Enable registration from this component?',
```

Comment:
I like documenting the props, but unless there is some kind of runtime support for this field (reading the Vue docs I couldn't find anything about "note" on props), we should instead document in comments, which can be stripped out during bundling.

We plan to use vue-styleguidist to document these, so follow [their conventions](https://github.com/vue-styleguidist/vue-styleguidist/blob/master/docs/Documenting.md). Unfortunately there's an upstream issue preventing us from doing this right now, which is tracked by #14, but once that's resolved we should just be able to flip a switch.